doc/FAQ.rst

```diff
@@ -59,49 +65,131 @@ Conversely, what Panel adds on top of Bokeh is full bidirectional communication
+|Supports fully interactive plots      | Yes             | Yes                  | Yes             | No?                | Yes                    | Yes?               |
++--------------------------------------+-----------------+----------------------+-----------------+--------------------+------------------------+--------------------+
+|Supports incremental plot updates     | Yes             | Yes                  | Yes             | No?                | Yes                    | Yes?               |
++--------------------------------------+-----------------+----------------------+-----------------+--------------------+------------------------+--------------------+
+|Fully usable in Jupyter               | Yes             | Yes                  | Yes, with       | No                 | No, only via iframe    | No                 |
+|                                      |                 |                      | jupyter_bokeh   |                    |                        |                    |
++--------------------------------------+-----------------+----------------------+-----------------+--------------------+------------------------+--------------------+
+|Supports static HTML export           | Yes             | Not without a special| Yes             | No?                | No                     | No?                |
+|(for reports, websites, docs)         |                 | embedding procedure  |                 |                    |                        |                    |
++--------------------------------------+-----------------+----------------------+-----------------+--------------------+------------------------+--------------------+
+|Supports Matplotlib plots             | Yes             | Yes                  | No              | Yes                | With a separate adapter| No                 |
++--------------------------------------+-----------------+----------------------+-----------------+--------------------+------------------------+--------------------+
+|Supports Bokeh plots                  | Yes             | Yes                  | Yes             | Yes                | With a separate adapter| No                 |
++--------------------------------------+-----------------+----------------------+-----------------+--------------------+------------------------+--------------------+
+|Supports Plotly plots                 | Yes             | Yes                  | No              | Yes                | Yes                    | No                 |
++--------------------------------------+-----------------+----------------------+-----------------+--------------------+------------------------+--------------------+
+|Supports R ggplot plots               | Yes             | No                   | No              | No                 | No                     | Yes                |
```

Comment:
Dash has R support (https://github.com/plotly/dashR), though I'm not sure of the status of the ggplot library.  How does panel support ggplot?  Could this row be labeled "R support" or something like that?

Reply:
DashR is something different; it's for how to use Dash from R.  This is for using R GGPlot from Python, for which Panel provides a specific RGGPlot pane: https://github.com/pyviz/panel/blob/master/panel/pane/plot.py#L97

That pane implementation acts as a guide for now to support other R plotting packages from Python without much effort, but it would take custom code to handle those, so we can't claim R support in general.  Not sure if there is a succinct way to express these nuances in a table...

Reply:
ahh, ok. I see. That makes sense then.